db/schema.rb

```diff
@@ -98,92 +98,92 @@
-    t.string   "content_id"
+    t.string   "ancestry",                 limit: 255
+    t.string   "contact_index_content_id", limit: 255
+    t.string   "content_id",               limit: 255
   end
 
   add_index "organisations", ["ancestry"], name: "index_organisations_on_ancestry", using: :btree
   add_index "organisations", ["slug"], name: "index_organisations_on_slug", unique: true, using: :btree
 
-  create_table "phone_numbers", force: true do |t|
-    t.integer  "contact_id"
-    t.string   "title"
-    t.string   "number"
-    t.text     "description"
-    t.text     "open_hours"
+  create_table "phone_numbers", force: :cascade do |t|
+    t.integer  "contact_id",                        limit: 4
+    t.string   "title",                             limit: 255
+    t.string   "number",                            limit: 255
+    t.text     "description",                       limit: 65535
+    t.text     "open_hours",                        limit: 65535
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "textphone"
-    t.string   "international_phone"
-    t.string   "fax"
-    t.string   "number_options_path"
-    t.string   "international_number_options_path"
-    t.text     "best_time_to_call"
+    t.string   "textphone",                         limit: 255
+    t.string   "international_phone",               limit: 255
+    t.string   "fax",                               limit: 255
+    t.string   "number_options_path",               limit: 255
+    t.string   "international_number_options_path", limit: 255
+    t.text     "best_time_to_call",                 limit: 65535
   end
 
   add_index "phone_numbers", ["contact_id"], name: "index_phone_numbers_on_contact_id", using: :btree
 
-  create_table "post_addresses", force: true do |t|
-    t.integer  "contact_id"
-    t.string   "title"
-    t.text     "description"
-    t.text     "address"
+  create_table "post_addresses", force: :cascade do |t|
+    t.integer  "contact_id",          limit: 4
+    t.string   "title",               limit: 255
+    t.text     "description",         limit: 65535
+    t.text     "address",             limit: 65535
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.text     "street_address"
-    t.string   "locality"
-    t.string   "region"
-    t.string   "postal_code"
-    t.string   "world_location_slug"
-    t.integer  "contact_type_id"
-    t.decimal  "latitude",            precision: 10, scale: 0
-    t.decimal  "longitude",           precision: 10, scale: 0
+    t.text     "street_address",      limit: 65535
+    t.string   "locality",            limit: 255
+    t.string   "region",              limit: 255
+    t.string   "postal_code",         limit: 255
+    t.string   "world_location_slug", limit: 255
+    t.integer  "contact_type_id",     limit: 4
+    t.decimal  "latitude",                          precision: 10
+    t.decimal  "longitude",                         precision: 10
```

Comment:
Actually, I think I'm going to drop this commit.  I don't think we should have these limits here at all, and in some cases they don't seem legit either.  `contact_id` on this table is an int(11) in my db, but it's setting `limit: 4` here.